/rosrider_node:
  ros__parameters:
    I2C_ENABLED: True
    ODOM_FRAME_ID: 'odom'
    BASE_FRAME_ID: 'base_footprint'
    BROADCAST_TF2: True
    PUB_ODOMETRY: True
    PUB_JOINTS: True
    PUB_DIAGNOSTICS: True
    ROS2RPI_CONFIG: 0x0F
    AUTOSYNC: True
    DEBUG: False
    CONFIG_FLAGS: 48
    UPDATE_RATE: 10
    ALLOWED_SKIP: 3
    PWM_DIV: 64
    DRIVE_MODE: 3
    OUTPUT_FILTER_TYPE: 1
    MONITOR_RATE: 100
    PWM_SCALE: 256
    PWM_FRQ: 50
    MAX_IDLE_SECONDS: 3600
    UPPER_LIMIT: 247
    INTEGRAL_LIMIT: 224
    ENCODER_PPR: 12
    RTC_TRIM: 0x7FFF
    LEFT_FORWARD_DEADZONE: 8
    LEFT_REVERSE_DEADZONE: 8
    RIGHT_FORWARD_DEADZONE: 8
    RIGHT_REVERSE_DEADZONE: 8
    GEAR_RATIO: 45.0
    WHEEL_DIA: 0.0685
    BASE_WIDTH: 0.1
    MAIN_AMP_LIMIT: 3.6
    BAT_VOLTS_HIGH: 15.0
    BAT_VOLTS_LOW: 6.0
    MAX_RPM: 90.0
    LEFT_AMP_LIMIT: 1.2
    RIGHT_AMP_LIMIT: 1.2
    LEFT_KP: 1.8
    LEFT_KI: 0.9
    LEFT_KD: 0.0
    RIGHT_KP: 1.8
    RIGHT_KI: 0.9
    RIGHT_KD: 0.0
    GAIN: 1.0
    TRIM: 0.032
    MOTOR_CONSTANT: 1.0
    I2C_ADDRESS: 0x3c
    ADCSYNC: True
    FASTADC: False
    CASCADED: False
    INA219_CAL: 8192
    TANH_DIV: 2.0
    SIGM_DIV: 10.0
    ADC_CS_DIV: 32
    CMD_VEL_TOPIC: "/cmd_vel"